styles/globals.css

```diff
@@ -33,26 +41,44 @@ body{
 }
 
 .nav {
- padding: 15px;
+ padding: 1rem;
  list-style-type: none;
  text-align: center;
 }
 
 .nav-list-container{
   display:flex;
-  padding: 3px;
+  padding: 0.188rem;
 }
 
 .nav li {
-  padding: 0 20px;
+  padding: 0 1.25rem;
   cursor: pointer;
 } 
 .nav li:hover {
-color: #B01E68;
+color: var(--color-tertiary);
 text-decoration: none;
 }
 
+.poi-container{
+  background-color: black;
+  width: 100vw;
+  height: 5vh;
+  padding: 0.188rem;;
```

Comment:
[gravel] I like to stay away from hyper-specific values like this, it makes scaling up and down on devices difficult

Reply:
fixed, kinda,  maybe there is a better way than the super low percents, I will look into it